packages/block-editor/src/components/inserter/category-tabs/index.js

```diff
@@ -33,7 +33,6 @@ function CategoryTabs( {
 
 	return (
 		<Tabs
-			className="block-editor-inserter__category-tabs"
```

Comment:
👍 

https://wpdirectory.net/search/01JAD1QEN6ZR91JYSVQPSHR50E

Reply:
I should have mentioned — applying a `className` to `Tabs` doesn't make any sense because `Tabs` doesn't render any element to the DOM (and doesn't accept a `className` prop at all)